experiments/a_b_guid.json

```diff
@@ -1,4 +1,5 @@
 {
+  "id": "feature_1",
```

Comment:
For these to be representative of how other systems would work, I think the `id` of these experiments would probably need to be the file path, ie. `experiments/a_b_guid.json`

Reply:
Ah, makes sense!